src/storage/mvcc/txn.rs

```diff
@@ -98,13 +98,14 @@ impl<S: Snapshot> MvccTxn<S> {
         lock_type: LockType,
         primary: Vec<u8>,
         short_value: Option<Value>,
+        lock_ttl: u64,
```

Comment:
so many arguments for this function, I think it can be refactored by @nrc ?

Reply:
:+1: on master rather than here, right?

Reply:
I think on master is enough